sql/core/src/main/scala/org/apache/spark/sql/UDFRegistration.scala

```diff
@@ -414,6 +418,84 @@ class UDFRegistration private[sql] (functionRegistry: FunctionRegistry) extends
   //////////////////////////////////////////////////////////////////////////////////////////////
 
   /**
```

Comment:
I can turn it on, but it would make the function less readable, especially for the following statements where it beyond line length limitation. 

```
case 14 => register(name, udf.asInstanceOf[UDF13[_, _, _, _, _, _, _, _, _, _, _, _, _, _]], returnType)
```